releasenotes/notes/0.19/refactor-set-qft-num-qubits-82e6df88448c2f22.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Refactored the :attr:`.QFT.num_qubits` setter to avoid potential problems when changing the number of qubits of the :obj:`.QFT` circuit. 
```

Comment:
I've actually dropped this release note - in practice there was no bug that could actually have been user-facing, because no matter what the value was, if it were truth-y it would be passed to `QuantumRegister` which would immediately error.  Erroring is the correct thing in this situation.  The old form actually provided a better error message than what we get now, because it does a type check, whereas non-numeric values now fail on `value > 0`.